postfix

```diff
@@ -58,4 +60,11 @@ fi
 ## run postfix
 /usr/sbin/postfix start-fg 2>&1 | rg --line-buffered -v "${LOG_FILTER_REGEX:-"^$"}" &
 
+# Reload config on USR1 signal
+postfix_reload {
```

Comment:
My preference would be to group all function definitions together at the top, and the trap commands together afterwards.